package.json

```diff
@@ -10,64 +10,62 @@
     "conf dir"
   ],
   "author": "Benjamin DANIEL <benjamin.daniel@neo9.fr>",
-  "license": "GNU",
+  "license": "MIT",
   "main": "dist/src/index.js",
   "types": "dist/src/index.d.ts",
   "files": [
     "dist/src"
   ],
   "scripts": {
     "build": "rimraf dist/ && tsc",
-    "lint": "tslint --project tsconfig.json --format verbose '{src,test}/**/*.ts'",
-    "lint:apply": "npm run format && tslint --fix --project tsconfig.json '{src,test}/**/*.ts'",
+    "lint": "eslint --config .eslintrc.yaml '{src,test}/**/*.ts'",
+    "lint:apply": "npm run format && npx eslint --fix '{src,test}/**/*.ts'",
     "format": "prettier --write '{,@(src|test)/**/}*.*'",
     "test:dev": "export NODE_ENV=test && TS_NODE_FILES=true ava --verbose --color --serial --watch",
-    "test": "npm run lint && npm run build && export NODE_ENV=test && nyc ava --verbose --color --serial && nyc report --reporter=html",
+    "test": "npm run lint && npm run build && export NODE_ENV=test && nyc ava --verbose --color --no-worker-threads --serial && nyc report --reporter=html",
     "coverage": "nyc report --reporter=text-lcov > coverage.lcov && codecov",
     "preversion": "npm test",
     "prepublishOnly": "npm run build",
     "release": "export BABEL_DISABLE_CACHE=1 && release-it --ci",
     "release:dry": "export BABEL_DISABLE_CACHE=1 && release-it --ci --dry-run=true",
-    "post:release": "echo Successfully released $npm_package_version"
+    "post:release": "echo Successfully released $npm_package_version",
+    "prepare": "husky install"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/neo9/n9-node-conf"
   },
   "engines": {
-    "node": ">=12.0.0"
+    "node": ">=16.0.0"
   },
   "dependencies": {
     "app-root-dir": "^1.0.2",
     "debug": "^4.3.1",
-    "fs-extra": "^10.0.0",
+    "fs-extra": "^11.1.1",
     "js-yaml": "^4.1.0",
     "lodash": "^4.17.21"
   },
   "devDependencies": {
```

Comment:
Some dependencies are not updated, exemple : 

    "@release-it/conventional-changelog": "^7.0.1",
    "codecov": "^3.8.3"
    "conventional-changelog-cli": "^4.0.0"